### PR TITLE
Fixes issue #14.

### PR DESCRIPTION
**Pull request by dougludlow**
Resolves the path of imports relative to the file requesting the import.

### CODE REVIEW

**Comment by screendriver** [comment]
Could you please update and check your pull request against pull request #22 which I merged a few minutes ago? Thank you.
By the way: thank you for the pull request :+1:

**Comment by dougludlow** [comment]
@screendriver sure thing. I'm thinking I'd like to ass some unit tests, do you have a preference on a testing framework? jasmine vs mocha?

**Comment by dougludlow** [comment]
I decided to go with tape.

**Comment by dougludlow** [comment]
@screendriver alrighty, it's all good to go.

**Comment by dougludlow** [comment]
@screendriver i found an issue affecting windows only and got that fixed. Think you'll have any time to take a look at the pr?

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// Place your settings in this file to overwrite default and user settings.
```
Comment (screendriver) [inline .vscode/settings.json]: This file should be added to `.gitignore`

File: src/resolve-path.js [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
     const { previous } = request;
-    if (previous !== 'stdin' && current.substr(0, 5) !== 'jspm:') {
-      current = url.resolve(previous, current);
-    }
+    let { current } = request;
     if (current.substr(0, 5) === 'jspm:') {
       current = current.replace(/^jspm:/, '') + '.scss';
       System.normalize(current)
-        .then(path => resolve(path.replace(/\.js$|\.ts$/, '')))
+        .then(p => resolve(p.replace(/\.js$|\.ts$/, '')))
```
Comment (screendriver) [inline src/resolve-path.js]: Why renaming `path` to `p`? I think it is more readable if variables have a readable name :wink:

File: src/sass-builder.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
 import fs from 'fs';
 import isEmpty from 'lodash/lang/isEmpty';
-import querystring from 'querystring';
```
Comment (screendriver) [inline src/sass-builder.js]: If you remove `querysring` please remove the dependency from `package.json` as well

File: src/sass-builder.js [hunk 4]
```diff
@@ -22,9 +22,9 @@ const escape = source => {
     .replace(/[\u2029]/g, '\\u2029');
 };
 
-const loadFile = path => {
+const loadFile = p => {
```
Comment (screendriver) [inline src/sass-builder.js]: Same here. Pleas do not rename the `path` variable for better reading

File: src/sass-builder.js [hunk 5]
```diff
@@ -34,11 +34,16 @@ const loadFile = path => {
...
 
-const parseUnescape = uri => {
-  // Node doesn't understand Windows' local file urls
-  // TODO: does not work in Linux / Mac OS X
-  // return (uri.match(/^file:\/\/\//)) ? uri.replace(/^file:\/\/\//, '') : querystring.unescape(url.parse(uri).path);
-  return querystring.unescape(url.parse(uri).path);
+const fromFileURL = a => {
+  let address = a.replace(/^file:(\/+)?/i, '');
+
+  if (!isWin) {
```
Comment (screendriver) [inline src/sass-builder.js]: ternary operator would be much shorter :wink:

File: src/sass-builder.js [hunk 6]
```diff
@@ -70,10 +76,10 @@ export default (loads, compileOpts) => {
 
   const compilePromise = load => {
     return new Promise((resolve, reject) => {
-      urlBase = load.address;
+      urlBase = path.dirname(load.address) + '/';
```
Comment (screendriver) [inline src/sass-builder.js]: Please use the new ES2015 template literals for better reading

File: src/sass-inject.js [hunk 7]
```diff
@@ -72,10 +75,10 @@ const compile = scss => {
 };
 
 export default load => {
-  urlBase = load.address;
-  const indentedSyntax = urlBase.endsWith('.sass');
+  urlBase = path.dirname(url.parse(load.address).pathname) + '/';
```
Comment (screendriver) [inline src/sass-inject.js]: Please use ES2015 string literals for better reading

**Comment by screendriver** [comment]
Hi @dougludlow,

thank you and sorry for the delayed answer. My desktop looks like traffic in rush hour at the moment...

For testing I would use mocha but tap is fine as well :wink: 

I've commented some of your files. It's only cosmetic. So far it looks good and all test are passing :+1: I will merge your pull request when you fix my little comments :wink:

File: src/resolve-path.js (hunk 2 above)
Comment (dougludlow) [inline src/resolve-path.js]: I had to rename it because I imported `path` above so I could call `path.dirname`.

Comment (dougludlow) [inline src/resolve-path.js]: But I could probably add a more readable name, regardless.

**Comment by dougludlow** [comment]
Good feedback, I'll get the pr updated. Thanks!

**Comment by dougludlow** [comment]
Alright, all done. Should I squash my commits or do you care?

**Comment by screendriver** [comment]
Thank you :+1: 
You don't have to squash. I will look into that next week after my holiday.

**Pull request merged**